mgmt_cli_test.py

```diff
@@ -173,21 +173,24 @@ def restore_backup_from_backup_task(self, mgr_cluster, backup_task, keyspace_and
                             keyspace_and_table_list=keyspace_and_table_list)
 
     # pylint: disable=too-many-arguments
-    def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspace1', tables_names=None,
+    def verify_backup_success(self, mgr_cluster, backup_task, ks_names: list = None, tables_names: list = None,
                               truncate=True, restore_data_with_task=False, timeout=None):
+        if ks_names is None:
+            ks_names = ['keyspace1']
         if tables_names is None:
             tables_names = ['standard1']
-        per_keyspace_tables_dict = {keyspace_name: tables_names}
+        ks_tables_map = {keyspace: tables_names for keyspace in ks_names}
         if truncate:
-            for table_name in tables_names:
-                self.log.info(f'running truncate on {keyspace_name}.{table_name}')
-                self.db_cluster.nodes[0].run_cqlsh(f'TRUNCATE {keyspace_name}.{table_name}')
+            for ks, tables in ks_tables_map.items():
+                for table_name in tables:
```

Comment:
nit (in general): I see a tendency to introduce small complications to save couple lines of code/reuse parts but decreasing readability. E.g. here instead of iterating over map, could just iterate over `ks_names` and `tables_names`.

That is my opinion only, just consider it.

Reply:
For this specific case, I'd prefer to keep it without changes.
It doesn't add much complexity and prevents us from further changes of these lines of code if we decide to rework the tables per keyspace assembling approach (for example, create different tables for different keyspaces).